app/views/group_offers/new.html.slim

```diff
@@ -0,0 +1,7 @@
+.row
```

Comment:
why do you put the h1 into a full width column?

h1 is a full width block element inside container on its own.